assets/material-design.js

```diff
@@ -0,0 +1,45 @@
+document.addEventListener("DOMContentLoaded", calculateDeliveryDays());
+
+function calculateDeliveryDays() {
+    let deliveryTime = document.querySelectorAll("[data-delivery-time]")[0].dataset.deliveryTime;
```

Comment:
Your code assumes that calls will always find expected elements. You might want to include error handling:

`if (!deliveryTime) {
        console.error("Delivery time element not found.");
        return;
    }`

Reply:
That's a good idea. I see that I don't instinctively create this type of exception handling yet